### PR TITLE
[TW-6556] Add variant selector

### PR DESCRIPTION
**Pull request by NathanDeVuono**
### Notes
This PR does not actually hook the variants up to anything, or do any filtering on variants other than getting a union of all the categories the product is in.

Pack sizes etc will be handling in coming PRs.

And then at some point I'm gonna have to figure out how to wire it all together for the Add to Cart button 😅 

### Screenshots
Mobile:
image

Desktop:
image

### CODE REVIEW

File: components/product/ProductView/ProductView.tsx [hunk 1]
```diff
@@ -64,7 +65,7 @@ const ProductView: FC<Props> = ({ product }) => {
       setLoading(false)
     }
   }
-  console.log(productDetails)
+  console.log(variants)
```
Comment (tomasztile) [inline components/product/ProductView/ProductView.tsx:68]: remove this eventually I guess

File: components/product/ProductView/ProductView.tsx [hunk 2]
```diff
@@ -100,6 +102,11 @@ const ProductView: FC<Props> = ({ product }) => {
   const getOGPrice = ({ salePrice }: ProductPrices) =>
     salePrice ? salePrice.value : undefined
 
+  const selectVariant = (e) => console.log(e.target.attributes.getNamedItem('value').value)
+    // (window.location.href = `/product${
```
Comment (tomasztile) [inline components/product/ProductView/ProductView.tsx:106]: Remove dead code

**Review by tomasztile — APPROVED** [review]

File: components/product/ProductView/ProductView.tsx (hunk 2 above)
Comment (NathanDeVuono) [reply components/product/ProductView/ProductView.tsx:106]: Good eyes, thanks

File: components/product/ProductView/ProductView.tsx (hunk 1 above)
Comment (NathanDeVuono) [reply components/product/ProductView/ProductView.tsx:68]: Yeah, it'll get removed when the PDP is closer to done

**Pull request merged**